PurchasesCoreSwift/LocalReceiptParsing/DataConverters/ArraySlice_UInt8+Extensions.swift

```diff
@@ -6,12 +6,12 @@
 import Foundation
 
 extension ArraySlice where Element == UInt8 {
-    func toUInt() -> UInt {
+    func toUInt() -> UInt64 {
```

Comment:
this isn't strictly necessary, but it doesn't do any harm other than parsing ever-so-slightly slower on 32-bit devices. 
The alternative was to write essentially the same method twice, once with `UInt` and the other one with `UInt64`.